tests/Acceptance/DeleteClientCommandTest.php

```diff
@@ -54,9 +54,9 @@ private function findClient(string $identifier): ?Client
             ;
     }
 
-    private function fakeAClient(string $identifier): Client
+    private function fakeAClient(string $name, string $identifier): Client
```

Comment:
Do we really need to make the name dynamic?

Reply:
probably not, i was just being careful when changing the tests to not break things :p but i guess this can be reverted ( same for the other )

Reply:
As it's not that important, let's keep it like that!